docs/reference.md

```diff
@@ -272,6 +272,37 @@ rendering of flows as graph are more gimmicks than productive. To execute any
 graph, one need to do a topological sort to transform it into a sequence, and
 sequences are more intuitive than graphs.
 
+### Retries
+
+<!-- TODO: Once Script retries implemented, move this part one header level higher -->
+
+Retries are an important tool to make your flows more robust, and more flexible.
+
+:::info 
+
+Retries are currently only available on Flow level, but we are working hard on
+bringing them to Scripts as well. Stay tuned!
```

Comment:
are only available at the top-level of a Flow but upon popular demand we will also bring them to the individual module level